src/main/java/seedu/address/logic/commands/LeaveCommand.java

```diff
@@ -0,0 +1,95 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_GROUP_NOT_FOUND;
+import static seedu.address.commons.core.Messages.MESSAGE_PERSON_NOT_FOUND;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.List;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.group.Group;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.shared.Title;
+
+
+
+/**
+ * Remove a person from a group existed in the address book.
+ */
+public class LeaveCommand extends Command {
+
+    public static final String COMMAND_WORD = "leave";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Removes a person from a group in the address book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_NAME + "GROUP\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "Derek Hardy "
+            + PREFIX_NAME + "GROUP_03";
+
+    public static final String MESSAGE_LEAVE_SUCCESS = "Person: %1$s removed from the group: %1$s";
+
+    private final Name personName;
+    private final Title groupName;
+    private Person matchedPersonByName;
+    private Group matchedGroupByName;
+
+    /**
+     * Creates an LeaveCommand to remove the specified {@code person} from {@code group}.
+     */
+    public LeaveCommand(Person person, Group group) {
+        requireAllNonNull(person);
+        this.personName = person.getName();
+        this.groupName = group.getTitle();
+        matchedPersonByName = null;
+        matchedGroupByName = null;
+    }
+
+    @Override
+    public CommandResult execute(Model model, CommandHistory history) throws CommandException {
+        requireNonNull(model);
+
+        List<Group> groupList = model.getGroupList();
+        for (Group group : groupList) {
+            Title title = group.getTitle();
+            if (title.equals(groupName)) {
+                matchedGroupByName = group.copy();
+                break;
+            }
+        }
+
+        List<Person> personList = model.getFilteredPersonList();
```

Comment:
getFiltered will cause bug when you add person to group after filtering the list